tests/unit/spec/base/mozilla-fxa-form.js

```diff
@@ -146,6 +148,62 @@ describe('mozilla-fxa-form.js', function() {
                 expect(form.querySelector('[name="utm_medium"]').value).toEqual('referral');
             });
         });
+
+        it('should pass through utm parameters from the URL to the form into the UITour', function() {
+            spyOn(window.Mozilla.Client, '_isFirefoxDesktop').and.returnValue(true);
+            spyOn(window.Mozilla.Client, '_getFirefoxVersion').and.returnValue('79.0');
+            spyOn(Mozilla.Client, 'getFirefoxDetails').and.callFake(function(callback) {
+                callback({
+                    'accurate': true,
+                    'distribution': undefined,
+                });
+            });
+            window.Mozilla.UITour = {
```

Comment:
Rather than overwriting these function I'd suggest using [sinon](https://sinonjs.org/releases/v9.0.2/stubs/) to stub things out. You can see an example of that here:

https://github.com/mozilla/bedrock/blob/master/tests/unit/spec/base/core-datalayer-page-id.js#L12-L16